From b74950d4f06bbfb91b2e68044147a226c15f4639 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Draszik?= <andre.draszik@jci.com>
Date: Mon, 30 Sep 2019 16:57:01 +0100
Subject: [PATCH] extmk: fix cross-compilation of external gems
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Note that I (André) didn't actually write this patch, I
only updated it so that git-am works.

Upstream-Status: Pending
Signed-off-by: André Draszik <andre.draszik@jci.com>
---
 ext/extmk.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/extmk.rb b/ext/extmk.rb
index d9c2417..da14c49 100755
--- a/ext/extmk.rb
+++ b/ext/extmk.rb
@@ -428,8 +428,8 @@ else
 end
 $ruby = [$ruby]
 $ruby << "-I'$(topdir)'"
+$ruby << "-I'$(top_srcdir)/lib'"
 unless CROSS_COMPILING
-  $ruby << "-I'$(top_srcdir)/lib'"
   $ruby << "-I'$(extout)/$(arch)'" << "-I'$(extout)/common'" if $extout
   ENV["RUBYLIB"] = "-"
 end
